packages/components/src/border-control/border-control/component.tsx

```diff
@@ -85,6 +91,8 @@ const UnconnectedBorderControl = (
 						__next36pxDefaultSize={ __next36pxDefaultSize }
 					/>
 					<UnitControl
+						label={ __( 'Border width' ) }
```

Comment:
Should the labels for the inner `UnitControl` and `RangeControl` be labelled differently when they are both for controlling the same thing i.e. "Border width"?

Reply:
I _think_ it's ok that they are the same, since the distinction is made by the screen reader when it says the role ("stepper" vs. "slider"). Though I'll defer to @afercia if there are better ways.

Reply:
I'll look to merge these improvements later today but am happy to tweak the labels further in a follow-up if desired.